src/extensions/askUserForParam.ts

```diff
@@ -116,6 +129,81 @@ export async function askForParam(paramSpec: Param): Promise<string> {
   return response;
 }
 
+async function promptSecret(
+  projectId: string,
+  instanceId: string,
+  paramSpec: Param
+): Promise<string> {
+  if (paramSpec.reconfiguring) {
```

Comment:
This feels like 2 functions pushed into one - one function that asks users if they want to reconfigure a secret, and another that asks users to create a new one. 

Consider splitting this into 2 functions 'promptReconfigureSecret' and 'promptCreateSecret', and calling those from here (or from askForParam)

Reply:
Refactored into two functions.